src/binder/bind/bind_file_scan.cpp

```diff
@@ -103,7 +124,23 @@ std::unique_ptr<BoundBaseScanSource> Binder::bindFileScanSource(const BaseScanSo
     for (auto i = 0u; i < bindData->columnTypes.size(); i++) {
         inputColumns.push_back(createVariable(bindData->columnNames[i], bindData->columnTypes[i]));
     }
-    auto info = BoundTableScanSourceInfo(func, std::move(bindData), inputColumns);
+
+    const bool ignoreErrors = config->getOption(CopyConstants::IGNORE_ERRORS_OPTION_NAME,
+        CopyConstants::DEFAULT_IGNORE_ERRORS);
+    column_id_t numWarningDataColumns = 0;
+    if (ignoreErrors) {
```

Comment:
shouldn't this be `if (!ignoreErrors)`?

Reply:
The intention is to output warnings only if `ignoreErrors=true` which is why the columns are set if `ignoreErrors=true`